include/bluetooth/mesh/main.h

```diff
@@ -1,5 +1,5 @@
 /** @file
- *  @brief Bluetooth Mesh Profile APIs.
+ *  @brief Provisioning APIs.
```

Comment:
I believe this should be reverted, the file contains both provisioning and core apis